### PR TITLE
Use upstream actions

### PR DESCRIPTION
**Pull request by theHamsta**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
   build:
+    name: Run tests
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: tree-sitter-grammars/actions/setup@main
+        with:
+          node-version: 20
+      - uses: tree-sitter-grammars/actions/test@main
```
Comment (ObserverOfTime) [inline .github/workflows/ci.yml]:
```suggestion
      - uses: tree-sitter/parser-test-action@v1.2
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,18 +1,28 @@
...
+      - binding.gyp
+
 jobs:
   build:
+    name: Run tests
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: tree-sitter-grammars/actions/setup@main
```
Comment (ObserverOfTime) [inline .github/workflows/ci.yml]:
```suggestion
      - uses: tree-sitter/parser-setup-action@v1.1
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -1,18 +1,28 @@
...
+        with:
+          node-version: 20
```
Comment (ObserverOfTime) [inline .github/workflows/ci.yml]:
```suggestion
          node-version: ${{vars.NODE_VERSION}}
```

File: .github/workflows/update-cpp-parser.yml [hunk 4]
```diff
@@ -1,44 +1,24 @@
...
-        run: |
-          git config user.name "GitHub"
-          git config user.email "noreply@github.com"
-          git commit -am "Update dependencies and regenerate parser" || echo 'No commit necessary!'
-          git clean -xf
-
-      - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v3
+          cache: npm
+      - uses: tree-sitter-grammars/actions/update@main
```
Comment (ObserverOfTime) [inline .github/workflows/update-cpp-parser.yml]:
```suggestion
      - uses: tree-sitter/parser-update-action@v1.1
```

File: .github/workflows/update-cpp-parser.yml [hunk 5]
```diff
@@ -1,44 +1,24 @@
...
-      - name: Commit changes
-        run: |
-          git config user.name "GitHub"
-          git config user.email "noreply@github.com"
-          git commit -am "Update dependencies and regenerate parser" || echo 'No commit necessary!'
-          git clean -xf
-
-      - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v3
+          cache: npm
```
Comment (ObserverOfTime) [inline .github/workflows/update-cpp-parser.yml:20]:
```suggestion
          cache: npm
          node-version: ${{vars.NODE_VERSION}}
```

**Review by ObserverOfTime — COMMENTED** [review]
The actions were moved to the tree-sitter org.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (theHamsta) [reply .github/workflows/ci.yml]: should this be fixed in the the tree-sitter-grammars action repo? I just copied from there

Comment (ObserverOfTime) [reply .github/workflows/ci.yml]: I archived it and linked to upstream.

**Pull request merged**